dashboards/resource_lakeview_dashboard_test.go

```diff
@@ -0,0 +1,181 @@
+package dashboards
+
+import (
+	"testing"
+
+	"github.com/databricks/databricks-sdk-go/service/dashboards"
+	"github.com/databricks/terraform-provider-databricks/qa"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestLakeviewDashboardCreate(t *testing.T) {
+	d, err := qa.ResourceFixture{
+		Fixtures: []qa.HTTPFixture{
```

Comment:
could we avoid using `Fixtures`, and use `MockWorkspaceClientFunc` instead for testing?